#goal definition
trajectory_custom_msgs/PointArray waypoints
---
#result definition
bool success
---
#feedback
float64 percentage
trajectory_custom_msgs/PointArray ee_pos